.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: python
+
+python:
+    - 2.7
+
+branches:
+    only:
+        - master
+        - develop
+
+env:
+    matrix:
+        - BACKEND_TEST=true JS_TEST=false INTEGRATION_TEST=false
+        - BACKEND_TEST=false JS_TEST=true INTEGRATION_TEST=false
+        - BACKEND_TEST=flase JS_TEST=false INTEGRATION_TEST=true
+
+before_install:
+    - pip install codecov
+    - export CHROME_BIN=chromium-browser
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+
+install:
+    - set -e
+    - pushd $TRAVIS_BUILD_DIR
+    - source scripts/setup.sh || exit 1
+    - source scripts/setup_gae.sh || exit 1
+
+    # Install Google App Engine
+    - mkdir -p $GOOGLE_APP_ENGINE_HOME
+    - curl --silent https://storage.googleapis.com/appengine-sdks/deprecated/1919/google_appengine_1.9.19.zip -o gae-download.zip
+    - unzip -qq gae-download.zip -d $TOOLS_DIR/google_appengine_1.9.19/
+    - rm gae-download.zip
+
+    # Install third party dependencies.
+    - if [[ $BACKEND_TEST == 'true' ]]; then bash -x scripts/install_third_party.sh; fi
```

Comment:
Yes, I left it out because third-party library installation is already handled in `scripts/run_integration_tests.sh`.